arangod/VocBase/Methods/Databases.cpp

```diff
@@ -417,9 +298,15 @@ arangodb::Result Databases::create(std::string const& dbName, VPackSlice const&
     return res;
   }
 
-  if (ServerState::instance()->isCoordinator()) {
+  if (ServerState::instance()->isCoordinator() /* REVIEW! && !localDatabase*/) {
```

Comment:
What is the purpose of this comment?

Reply:
I am not sure if local dbs of a coordinator need global ticks.